### ISSUE TITLE
error with data-ipa-cfg-mgr/hal

### ISSUE THREAD

**Issue by kj415j45** [zuk-devs/android_device_zuk_msm8996-common#20]
Log: 
```
[1067/1071] including vendor/qcom/opensource/data-ipa-cfg-mgr/Android.mk ...
build/make/core/base_rules.mk:260: error: vendor/qcom/opensource/data-ipa-cfg-mgr/hal: MODULE.TARGET.SHARED_LIBRARIES.liboffloadhal already defined by device/zuk/msm8996-common/data-ipa-cfg-mgr/hal.
18:51:28 ckati failed with: exit status 1
build/make/core/main.mk:21: recipe for target 'run_soong_ui' failed
make: *** [run_soong_ui] Error 1
```

occurred when building AEX 6.2, device tree:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<manifest>
    <project name="zuk-devs/android_device_zuk_z2_plus" path="device/zuk/z2_plus" remote="github" revision="9x-caf" />
    <project name="zuk-devs/android_device_zuk_msm8996-common" path="device/zuk/msm8996-common" remote="github" revision="9x-caf" />
    <project name="zuk-devs/android_vendor_zuk" path="vendor/zuk" remote="github" revision="9x-caf" />
    <project name="DD3Boh/android_kernel_zuk_msm8996" path="kernel/zuk/msm8996" remote="github" revision="lineage-16.0" />

    <project name="teamfirangi/linaro-7.3" path="prebuilts/gcc/linux-x86/aarch64/aarch64-linaro-7.3" remote="github" revision="master" />

    <remove-project path="packages/apps/Launcher3" name="platform_packages_apps_Launcher3" groups="pdk-fs" />
    <project name="AospExtended/platform_packages_apps_Launcher3" path="packages/apps/Launcher3" remote="github" revision="9.x" />

</manifest>
```

**Comment by RuRuTiaSaMa** [comment]
delete data-ipa-cfg-mgr dir to use local data-ipa-cfg-mgr at vendor/qcom/opensource/data-ipa-cfg-mgr

**Comment by RuRuTiaSaMa** [comment]
Or Insert a declaration
like:
USE_DEVICE_SPECIFIC_DATA_IPA_CFG_MGR := true
to build data-ipa-cfg-mgr from device tree

**Comment by RuRuTiaSaMa** [comment]
@kj415j45 you can close this issue later .....

**Comment by DD3Boh** [comment]
Issue fixed with f67441ec88091461022c993502dd26216a4f9c10
